config/locales/en/stats.yml

```diff
@@ -14,7 +14,7 @@ en:
     total: "Total"
     geozone: "District"
     no_demographic_data:
-      zero: ""
+      zero: "* There are %{count} participants."
```

Comment:
The goal here was not to show any message for the `zero` case, according to the [commit message](https://github.com/consul/consul/commit/e146fafb60) :thinking:.

> If there's demographic data for all participants, it doesn't make sense to show the message.
>
> We're using translations instead of an `if` in the view because the text is also different when there's only one participant. In some languages the text might also be different depending on how many people with no
demographic data participated.
>
> Another possibility would be to use an `if` in the view so we don't display an empty paragraph when the cont is zero, and then using translation for `one` and `other`. I haven't gone that way because I thought the logic would be more complex and the benefits wouldn't be that great.

Obviously whoever wrote this comment (it was me :smile:) was ignorant and didn't know there'd be issues with translations.

So I guess we should add a condition to the budget  and poll stats views so the message isn't shown in this case :thinking:.

Also note the current message would need to be changed since "There are zero participants" isn't true; it's zero participants with no demographic data.

```suggestion
      zero: "* There is demographic for all participants."
```

By the way, if we change the English text, shall we change the Spanish one too?

Reply:
Hi, @javierm, now we have the condition I think there is no need to keep that key anymore 🤔. Can we delete that entry? or am I losing something?

Reply:
Hadn't thought about that. I think we can delete it, yes :+1:.